src/Pipelines/multisample.php

```diff
@@ -263,6 +275,58 @@ function extract_info($format, $data)
 			$args[] = "-mode dragen";
 			$parser->execTool("Tools/merge_gvcf.php", implode(" ", $args));
 		}
+		elseif($use_deepvar) //calling with DeepVariant with gVCF file creation
+		{
+			$deepvar_gvcfs = array();
+			foreach($local_bams as $local_bam) // DeepVariant calling for each bam with gVCF output
+			{
+				$deepvar_vcf = $parser->tempFile("_deepvar.vcf.gz");
+				$deepvar_gvcf = $parser->tempFile("_deepvar.gvcf");
+
+				$args = [];
+
+				if ($is_wes || $is_wgs)
+				{
+					$args[] = "-model_type ".$sys['type'];
+				}
+				else
+				{
+					trigger_error("The usage of DeepVariant is limited to WGS or WES short-read data! Different type '".$sys['type']."' detected in $system.", E_USER_ERROR);
+				}
+
+				$args[] = "-bam ".$local_bam;
+				$args[] = "-out ".$deepvar_vcf;
+				$args[] = "-gvcf ".$deepvar_gvcf;
+				$args[] = "-build ".$sys['build'];
+				$args[] = "-threads ".$threads;
+				$args[] = "-target ".$sys['target_file'];
+				$args[] = "-target_extend 200";
+
+				$parser->execTool("Tools/vc_deepvariant.php", implode(" ", $args));
+				
+				$deepvar_gvcfs[] = $deepvar_gvcf;
+			}
+
+			//Merge gVCFs with GLnexus
+			$glnexus_tmp = $parser->tempFolder().getmypid();
+			$pipeline = array();
+
+			// GLnexus args
+			$args = array();
+			$args[] = "--dir ".$glnexus_tmp;
+			if ($is_wes) $args[] = "--config DeepVariantWES";
+			else $args[] = "--config DeepVariantWGS";
+			$args[] = implode(" ", $deepvar_gvcfs);
+
+			// Merge pipeline
+			$pipeline[] = ["", $parser->execApptainer("glnexus", "glnexus_cli", implode(" ", $args), [], [], true)];
+			$pipeline[] = ["", $parser->execApptainer("glnexus", "bcftools view", "", [], [], true)];
+			$pipeline[] = ["bgzip", "-@ -c > $vcf_all"];
+
+			$parser->execPipeline($pipeline, "GLnexus gVCF merging");
+			trigger_error("Temporary folder used as GLnexus.DB folder: $glnexus_tmp", E_USER_NOTICE);
+			$parser->exec("rm", "-rf $glnexus_tmp");
```

Comment:
not necessary.
Folder created with $parser->tempFolder() is automatically deleted.